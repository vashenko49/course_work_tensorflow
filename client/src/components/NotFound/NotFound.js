import React, { Component } from 'react';
import './NotFound.scss';
import { Typography } from '@material-ui/core';

class NotFound extends Component {
  render() {
    return (
      <div className="not-found">
        <Typography variant={'h2'}>404</Typography>
        <Typography variant={'h6'}>We can't find the page you are looking for.</Typography>
        <Typography variant={'h6'}>Sorry for the inconvenience</Typography>
      </div>
    );
  }
}

export default NotFound;
