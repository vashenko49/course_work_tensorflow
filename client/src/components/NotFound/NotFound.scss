.not-found{
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 207px 0;
  font-weight: 500;
}