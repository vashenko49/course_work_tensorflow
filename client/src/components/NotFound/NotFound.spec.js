import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import { NotFound } from './NotFound';

describe('<NotFound />', () => {
  let shallow;
  beforeAll(() => {
    shallow = createShallow();
  });

  it('should work', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
