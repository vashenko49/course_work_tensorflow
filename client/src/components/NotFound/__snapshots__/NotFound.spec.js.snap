// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should work 1`] = `
<div
  className="not-found"
>
  <WithStyles(ForwardRef(Typography))
    variant="h2"
  >
    404
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    variant="h6"
  >
    We can't find the page you are looking for.
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    variant="h6"
  >
    Sorry for the inconvenience
  </WithStyles(ForwardRef(Typography))>
</div>
`;
